show/main.py

```diff
@@ -831,64 +754,133 @@ def ipv6():
     pass
 
 #
-# 'prefix-list' subcommand ("show ipv6 prefix-list")
+# 'show route-map' command ("show route-map")
 #
+if routing_stack != "framewave":
+    @cli.command('route-map')
+    @click.argument('route_map_name', required=False)
+    @click.option('--verbose', is_flag=True, help="Enable verbose output")
+    def route_map(route_map_name, verbose):
+        """show route-map"""
+        cmd = 'sudo {} -c "show route-map'.format(constants.RVTYSH_COMMAND)
+        if route_map_name is not None:
+            cmd += ' {}'.format(route_map_name)
+        cmd += '"'
+        run_command(cmd, display_cmd=verbose)
 
-@ipv6.command('prefix-list')
-@click.argument('prefix_list_name', required=False)
-@click.option('--verbose', is_flag=True, help="Enable verbose output")
-def prefix_list(prefix_list_name, verbose):
-    """show ip prefix-list"""
-    cmd = 'sudo {} -c "show ipv6 prefix-list'.format(constants.RVTYSH_COMMAND)
-    if prefix_list_name is not None:
-        cmd += ' {}'.format(prefix_list_name)
-    cmd += '"'
-    run_command(cmd, display_cmd=verbose)
 
+    #
+    # 'show ip interfaces' command
+    #
+    # Display all interfaces with master, an IPv4 address, admin/oper states, their BGP neighbor name and peer ip.
+    # Addresses from all scopes are included. Interfaces with no addresses are
+    # excluded.
+    #
+    @ip.command()
+    @multi_asic_util.multi_asic_click_options
+    def interfaces(namespace, display):
```

Comment:
>def interfaces [](http://example.com/codeflow?start=4&length=14)

It's tricky to define functions inside if-branch. Could you move it back to top level?